queries.sql

```diff
@@ -1,7 +1,22 @@
 -- Part 1: Test it with SQL
 
+-- Columns [Data type]
+-- id [INT]
+-- employer [VARCHAR(255)]
+-- name [VARCHAR(255)]
+-- skills [VARCHAR(255)]
+
+
 -- Part 2: Test it with SQL
 
+-- SELECT *
+-- FROM employer
+-- WHERE (location = "St. Louis City");
+
+
 -- Part 3: Test it with SQL
 
+--DROP TABLE job
```

Comment:
```suggestion
--DROP TABLE job;
```
This query is also correct! It's only missing a semi-colon at the end of the command.